src/next-js.ts

```diff
@@ -48,25 +52,63 @@ export function connectWrapper<Req extends Request, Res extends Response>(
     }).then(next);
 }
 
-export function toNodeHandler(config: TidewaveConfig = DEFAULT_CONFIG): NextJsHandler {
-  // we don't need any `next` request handler
-  const next: () => void = () => {};
+function notfound(req: NextApiRequest, res: NextApiResponse): void {
+  return res.status(404).json({ message: `Route not found: ${req.method} ${req.url}` });
+}
+
+const NEXT_HANDLERS: Record<string, (req: Request, res: Response, next: NextFn) => Promise<void>> =
```

Comment:
My suggestion: let's move this to http like this:

```
HANDLERS = {
  mcp: handleMcp,
  shell: handleShell
}
```

And then we use it when mounting the Vite server and Next.JS middleware, so we only define them in a single place. :)